pkg/build/docker_go.go

```diff
@@ -578,8 +582,11 @@ ARG RUNTIME_IMAGE=busybox:1.31.1-glibc
 #:::
 FROM ${BUILD_BASE_IMAGE} AS builder
 
+# PLAN_PATH is the path of our test's source code.
+ARG PLAN_PATH
```

Comment:
Just to confirm. This defaults to empty string, right?

Reply:
It defaults to `./` (in `docker_generic.go`), this way `Dockerfile` authors don't need to change their code on move.